torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -601,9 +605,7 @@ def _pre_backward_hook(
     if _handles_key and state._ran_pre_backward_hook.get(_handles_key, False):
         return
 
-    with torch.autograd.profiler.record_function(
-        "FullyShardedDataParallel._pre_backward_hook"
-    ):
+    with torch.profiler.record_function("FullyShardedDataParallel._pre_backward_hook"):
```

Comment:
@rohan-varma  I was wondering what is the difference between `torch.autograd.profiler...` vs. `torch.profiler...` and why we changed this here.